src/gui/qgsmapcanvas.h

```diff
@@ -116,6 +116,10 @@ class GUI_EXPORT QgsMapCanvas : public QGraphicsView
     //! Destructor
     ~QgsMapCanvas();
 
+    //! sets the level of magnification
+    //! @note added in 2.16
```

Comment:
Can you expand these docs a bit? It's quite clear that this sets the magnification, but may not be immediately obvious exactly what this means...